.github/workflows/build.yaml

```diff
@@ -1,49 +1,43 @@
-name: Update devs api image
+name: Build, tag, and publish
 env:
   # This must be defined for the bash redirection
   GOOGLE_APPLICATION_CREDENTIALS: 'jade-dev-account.json'
   # This must be defined for the bash redirection
   GOOGLE_SA_CERT: 'jade-dev-account.pem'
 on:
   workflow_dispatch: {}
+  pull_request:
+    types:
+      - opened
+      - reopened
+      - synchronize  # https://github.community/t/what-is-a-pull-request-synchronize-event/14784/2
```

Comment:
Is there a great benefit in being more specific here? Our current PR build workflow uses `pull_request:` without specifying the events the workflow is limited to, and seems to be working fine.

Reply:
My inclination is to have less code when possible and lean on defaults if they make sense, so I would rather not having the event types listed out.

Reply:
Done